enterprise/backup/src/test/java/org/neo4j/backup/OnlineBackupCommandCcIT.java

```diff
@@ -0,0 +1,228 @@
+    public static List<String> recordFormats()
+    {
+        return Arrays.asList( Standard.LATEST_NAME, HighLimit.NAME );
+    }
+
+    @Before
+    public void initialiseBackupDirectory()
+    {
+        backupDir = testDirectory.directory( "backups" );
+    }
+
+    @Test
+    public void backupCanBePerformedOverCcWithDefaultPort() throws Exception
+    {
+        assumeFalse( SystemUtils.IS_OS_WINDOWS );
+
+        Cluster cluster = startCluster( null, false, null, recordFormat );
+        createSomeData( clusterDatabase( cluster ) );
+        assertEquals(
+                0,
+                runBackupToolFromOtherJvmToGetExitCode( "--from", ip,
+                        "--cc-report-dir=" + backupDir,
+                        "--backup-dir=" + backupDir,
+                        "--name=defaultport" ) );
+        assertEquals( DbRepresentation.of( clusterDatabase( cluster ) ), getBackupDbRepresentation( "defaultport" ) );
+
+        createSomeData( clusterDatabase( cluster ) );
+        assertEquals(
+                0,
+                runBackupToolFromOtherJvmToGetExitCode( "--from", ip,
+                        "--cc-report-dir=" + backupDir,
+                        "--backup-dir=" + backupDir,
+                        "--name=defaultport" ) );
+        assertEquals( DbRepresentation.of( clusterDatabase( cluster ) ), getBackupDbRepresentation( "defaultport" ) );
+    }
+
+    @Test
+    public void backupCanBePerformedOverCcWithCustomPort() throws Exception
+    {
+        assumeFalse( SystemUtils.IS_OS_WINDOWS );
+        String customAddress = ip + ":" + 19911;
+
+        Cluster cluster = startCluster( 19911, false, null, recordFormat );
+        assertEquals(
+                1,
+                runBackupToolFromOtherJvmToGetExitCode( "--cc-report-dir=" + backupDir,
+                        "--backup-dir=" + backupDir,
+                        "--name=defaultport" ) );
+        assertEquals(
+                0,
+                runBackupToolFromOtherJvmToGetExitCode( "--from", customAddress,
+                        "--cc-report-dir=" + backupDir,
+                        "--backup-dir=" + backupDir,
+                        "--name=defaultport" ) );
+        assertEquals( DbRepresentation.of( clusterDatabase( cluster ) ), getBackupDbRepresentation( "defaultport" ) );
+
+        createSomeData( clusterDatabase( cluster ) );
+        assertEquals(
+                0,
+                runBackupToolFromOtherJvmToGetExitCode( "--from", customAddress,
+                        "--cc-report-dir=" + backupDir,
+                        "--backup-dir=" + backupDir,
+                        "--name=defaultport" ) );
+        assertEquals( DbRepresentation.of( clusterDatabase( cluster ) ), getBackupDbRepresentation( "defaultport" ) );
+    }
+
+    @Test
+    @Ignore( "Need to resolved this - causal clustering should have the backup protocol disabled" )
+    public void backupCanNotBePerformedOverHa() throws Exception
```

Comment:
The test does not look like it's starting an HA cluster.

Reply:
The test starts a causal cluster. There are 2 separate IT classes for OnlineBackupCommand:
OnlineBackupCommand*Cc*IT and OnlineBackupCommand*Ha*IT
This test is ignored, because we are expecting the new behaviour for CC to not have the backup protocol enabled (at all). therefore this test should fail.